tests/test_sftp.py

```diff
@@ -554,6 +554,32 @@ def progress_callback(x, y):
         os.unlink(localname)
         sftp.unlink(sftp.FOLDER + "/bunny.txt")
 
+    def test_get_without_prefetch(self, sftp):
+        """
+        Create a 4MB file. Verify that pull works without prefetching
+        using a lager file.
+        """
+
+        fd, localname = mkstemp()
+        os.close(fd)
+
+        with open(localname, 'wb') as f:
+            num_chars = 1024 * 1024 * 4
+            f.write(b'0' * num_chars)
+
+        sftp.put(localname, sftp.FOLDER + "/dummy_file")
+
+        os.unlink(localname)
+        fd, localname = mkstemp()
+        os.close(fd)
+
+        sftp.get(sftp.FOLDER + "/dummy_file", localname, prefetch=False)
+
+        assert os.stat(localname).st_size == 4194304
```

Comment:
I'm not expert here, but looks good to me.

For readability, and to avoid the magic numbers/strings, maybe refactor:

1) The number of characters to e.g. `NUM_CHARS` at the test function scope, and then use it both with the `f.write(b'0' * NUM_CHARS)` above and also in this assertion for the file size; and,

2) The `sftp.FOLDER + "/dummy_file"` to e.g. `STFP_FILENAME` at test function scope?

Reply:
> I'm not expert here, but looks good to me.
> 
> For readability, and to avoid the magic numbers/strings, maybe refactor:
> 
> 1. The number of characters to e.g. `NUM_CHARS` at the test function scope, and then use it both with the `f.write(b'0' * NUM_CHARS)` above and also in this assertion for the file size; and,
> 2. The `sftp.FOLDER + "/dummy_file"` to e.g. `STFP_FILENAME` at test function scope?

me neither but I hope with this update I was able to adjust the readability according to your suggestions

Reply:
Yep! That looks good to me. I've flagged the PR for @bitprophet's attention; he'll be the one to set the plan for a merge &c.

Thanks for the contribution!